pybossa/jobs.py

```diff
@@ -979,7 +987,7 @@ def export_tasks(current_user_email_addr, short_name,
                 mail_dict['attachments'] = [Attachment(filename, "application/zip", content)]
 
             current_app.logger.info(
-                'Tasks exported successfully - Project: {0}'.format(project.name))
+                'Tasks export completed - Project: {0}'.format(project.name))
```

Comment:
It would be ok provided someone is not using this in their dashboards that will stop working. Any reason to change this?

Reply:
There are multiple ways in which tasks exported can be shipped; as email attachment or upload to s3. I think it would make sense to record which method has been followed. Meaning
1. Log after [line 970](https://github.com/bloomberg/pybossa/blob/334d05b8ae14180a2edab813955f4f706fa1ef37/pybossa/jobs.py#L970) that export has failed due to exceeding max size
2. Task exported attached to email at [line 986](https://github.com/bloomberg/pybossa/blob/334d05b8ae14180a2edab813955f4f706fa1ef37/pybossa/jobs.py#L986)
3. Remove [this log](https://github.com/bloomberg/pybossa/blob/334d05b8ae14180a2edab813955f4f706fa1ef37/pybossa/jobs.py#L987-L988) as each if-elif-else block has their associated logs covered.

Also, it would be good if each if-elif-else block can follow a standard pattern of logging 
Ex
a. `Task export project id <project-id>. Export exceeded max file size <max-size>, actual size <actual-size>` 
b. `Task export project id <project-id>. Exported file uploaded to s3 <s3-url>`
c. `Task export project id <project-id>. Exported file attached to email to send`